ocaml/xenopsd/lib/xenops_server.ml

```diff
@@ -1864,10 +1864,17 @@ let rec perform_atomic ~progress_callback ?subtask:_ ?result (op : atomic)
           (Xenops_task.id_of_handle t)
           (List.length atoms) description
       in
+      let parallel_id_with_tracing =
+        match Xenops_task.tracing t with
+        | Some t ->
+            parallel_id ^ "\x00" ^ t
```

Comment:
Can we put this in a function in xenops_task? this is leaking an implementation detail.
Using `Xenops_task.( parallel_id // id)`, similar to how FIlenmae.concat is used would be quite understandable

Reply:
I can do that?

Reply:
I don't see why not, just implement the function in xenops_task as:
```ocaml
let ( // ) parent id = parent ^ "\x00" ^ id
```